mri/operators/fourier/non_cartesian.py

```diff
@@ -459,7 +459,7 @@ def _op(self, data):
             np.sqrt(self.num_slices / self.acq_num_slices)
 
     def op(self, data):
-        """ This method calculates Fourier transform.
+        """ This method computes Fourier transform.
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text

```suggestion
        """Compute the Fourier transform.
```